launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/data/Channel.java

```diff
@@ -110,14 +109,13 @@ public String getAuthor() {
   }
 
   /**
-  * Add an item to a channel.
-  */
+   * Add an item to a channel.
+   */
   public void addItem(String podcastTitle, String description, String language, String email, String mp3Link) {
     Item item = new Item(podcastTitle, description, language, email, mp3Link);
     if (this.items == null) {
       this.items = new ArrayList<>();
     }
```

Comment:
this.items = new ArrayList<>() should be put in the constructor so you never have to worry about it being null